bokehjs/src/lib/models/glyphs/marker.ts

```diff
@@ -162,6 +163,7 @@ export namespace Marker {
   export type Props = XYGlyph.Props & {
     size: p.DistanceSpec
     angle: p.AngleSpec
+    point_hittest_dialation:  p.Property<number>
```

Comment:
It should be `dilation`. Also keep consistent styling (i.e. single space).

Reply:
just "dilation" seems a bit over-broad, I think it could easily be mis-understood by users to be intended for some sort of visual dilation, e.g. I think this should be more specific, at least `hit_dilation`

Reply:
I meant `point_hittest_dialation` -> `point_hittest_dilation`. But a shorter name would be nice, if possible.